vacasa/connect/connect.py

```diff
@@ -1480,6 +1480,15 @@ def update_contact_finances_payload(self, contact_id, params: dict):
         url = f"{self.endpoint}/v1/contacts/{contact_id}/finances"
         self._patch(url, json={'data': {'attributes': params}}, headers=self._headers())
 
+    def get_language_list(self):
+        """
+
+        Returns: List of languages from Connect
+
+        """
```

Comment:
I would change this to: `"""Get a list of languages from Connect"""`, but that's just personal preference.